src/pygeoprocessing/geoprocessing.py

```diff
@@ -574,6 +578,9 @@ def align_and_resize_raster_stack(
             name string as the first element and a GDAL creation options
             tuple/list as the second. Defaults to a GTiff driver tuple
             defined at geoprocessing.DEFAULT_GTIFF_CREATION_TUPLE_OPTIONS.
+        osr_axis_mapping_strategy (OSR object): The OSR Axis Mapping Strategy 
+            as the first element. Defaults to a GIS friendly Lon,Lat order 
```

Comment:
It's not an OSR object, rather just an integer, can you change the type to (int)? Also the mention of "first element" sounds like it's a tuple. Could this read simply as:

```
osr_axis_mapping_strategy (int): OSR axis mapping strategy for `SpatialReference` objects. Defaults to 
    `geoprocessing.DEFAULT_OSR_AXIS_MAPPING_STRATEGY`. This parameter should not be changed unless you 
    know what you are doing.
```

Reply:
Done in [2eba123](https://github.com/natcap/pygeoprocessing/pull/31/commits/2eba123c585913539377eeaa05ebccdf13990a80)